source/utilities.tex

```diff
@@ -1957,6 +1957,15 @@
 \item \tcode{is_constructible_v<$\tcode{A}_{ik}$, $\cv{}_i\;\tcode{A}_{ik}$\&\&> == true}.
 \end{itemize}
 
+\pnum
+\returns
+A \tcode{tuple} object constructed by initializing the ${k_i}^\text{th}$
+type element $\tcode{e}_{ik}$ in \tcode{$\tcode{e}_i$...} with
+\begin{codeblock}
+get<@$k_i$@>(std::forward<@$\tcode{T}_i$@>(@$\tcode{tp}_i$@))
+\end{codeblock}
+for each valid $k_i$ and each group $\tcode{e}_i$ in order.
+
```

Comment:
This uses the _`e`<sub>i</sub>_ defined in the _Remarks:_ below. Should it be at the top of the description, like elsewhere?

Reply:
If it would only define e_i has a tool, then yes. But it's also specifying requirements on the result type CTypes.

Reverted.